src/controllers/users.js

```diff
@@ -1,11 +1,48 @@
 /* Get all users from Firebase */
 import * as admin from 'firebase-admin';
 
-export const getUsers = async (req, res) => {
-  // Use Firebase functions to get all user objects
-  // This will allow for selecting individual users and updating their user roles
-  const result = await admin.default.auth().listUsers();
-  const users = result.users.map((userRecord) => userRecord.toJSON());
+const formatUser = (user) => {
+  const customClaims = (user.customClaims || { role: '' });
+  const role = customClaims.role ? customClaims.role : '';
+  return {
+    uid: user.uid,
+    id: user.uid,
+    email: user.email || '',
+    displayName: user.displayName || '',
+    role,
+    lastSignInTime: user.metadata.lastSignInTime,
+    creationTime: user.metadata.creationTime,
+  };
+};
+
+/* Grab all users in the Firebase database */
+export const getUsers = async (_, res) => {
```

Comment:
@ebubae Turns out that we need to have these functions in the API instead of the editor platform so react-admin can automatically make its pre-built `GET` requests for all users or just a single user.